src-tauri/src/main.rs

```diff
@@ -57,6 +62,17 @@ async fn list_projects(state: tauri::State<'_, ApplicationState>) -> Result<Vec<
         .map_err(|e| e.to_string())
 }
 
+#[tauri::command]
+async fn create_project(
+    state: tauri::State<'_, ApplicationState>,
+    name: &str,
+    description: Option<&str>,
+) -> Result<models::Project, String> {
```

Comment:
This is related to a comment I just posted in my PR, but I was wondering what would be the best place to store the cloud project id - can it be a part of the `Project` model? If so, it would probably be good to add it as an optional argument here